core/commands/files/files.go

```diff
@@ -137,11 +137,11 @@ func statNode(ds dag.DAGService, fsn mfs.FSNode) (*Object, error) {
 
 var FilesCpCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "copy files into mfs",
```

Comment:
**to be clear**: that hierarchy is in raw IPLD objects, and not unixfs directories.